apps/dashboard/src/constants/urls.ts

```diff
@@ -10,3 +10,6 @@ export const THIRDWEB_EWS_API_HOST =
 export const THIRDWEB_PAYMENTS_API_HOST =
   process.env.NEXT_PUBLIC_THIRDWEB_PAYMENTS_API_HOST ||
   "https://payments.thirdweb.com";
+
+export const THIRDWEB_PAY_DOMAIN =
+  process.env.NEXT_PUBLIC_PAY_URL || "pay.thirdweb-dev.com";
```

Comment:
make sure this is set in vercel env vars, especially since we default to dev here

Reply:
Good callout - I thought because this env was already being used elsewhere - this is setup in Vercel. But it was not, adding it